CONTRIBUTING.md

```diff
@@ -223,7 +223,7 @@ Other coding conventions include:
 * Properties in `GitMacheteBundle.properties` that use HTML should be wrapped in tags `<html>` ... `</html>`.
   Additionally, their keys should have a `.HTML` suffix.
 * `@Tainted` and `@Untainted` annotations are used in the context of method parameters that may or may not use HTML. Those annotated with `@Untainted` should not contain HTML tags, whereas values annotated with
-  `@Tainted` can contain HTML (but they don't have to).
+  `@Tainted` can contain HTML (but they don't have to). Please note that if needed, you might need to use the` @Tainted` or `@Untainted` tags from the Checker Framework, rather than the Java Annotations ones.
```

Comment:
```suggestion
  `@Tainted` can contain HTML (but they don't have to). Please note that you need to use the` @Tainted` or `@Untainted` annotations from the Checker Framework, rather than the Java Annotations ones.
```

Reply:
Actually, pls use FQCNs for clarity